src/App.css

```diff
@@ -469,3 +469,66 @@ button:enabled:hover {
   color: #1c2334;
   text-decoration: underline;
 }
+
+/*--------------------------------------------------------------------------*\
+  Settings
+\*--------------------------------------------------------------------------*/
+.settings__title {
+  text-align: center;
+}
+
+.settings__title + * {
+  margin-top: 1rem;
+}
+
+.settings__explanation {
+  text-align: center;
```

Comment:
This all looks sharp but my brain is messing with me in mobile, the list text dashes push the text a little off center, which is not a deal breaker, but I keep wanting to nudge it to the left a bit.
<img width="343" alt="Screen Shot 2021-08-17 at 6 12 16 PM" src="https://user-images.githubusercontent.com/2355543/129820907-cf2e7fd8-29dd-4707-b2c6-ffdffc5c1c86.png">

Reply:
100% agree - the dashes have been removed